app/views/layouts/_meta.html.haml

```diff
@@ -1,6 +1,7 @@
 %head
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta property="og:image" content="meta_image.png" />
```

Comment:
this is a small thing, but perhaps we could rename this something like growstuff_logo_ogimage.png or something?  "meta_image" doesn't really tell us what it's for.